src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-trash-move-modal/rbd-trash-move-modal.component.html

```diff
@@ -0,0 +1,60 @@
+<cd-modal>
+  <ng-container i18n
+                class="modal-title">Move an image to trash</ng-container>
+
+  <ng-container class="modal-content">
+    <form name="moveForm"
+          class="form"
+          #formDir="ngForm"
+          [formGroup]="moveForm"
+          novalidate>
+      <div class="modal-body">
+        <p>
+          <ng-container i18n>To move</ng-container>
```

Comment:
Please apply the following patch to fix the text issue.

![auswahl_002](https://user-images.githubusercontent.com/1897962/45826114-0d8a2700-bcf4-11e8-97c3-3f5c91be5486.png)

```
--- src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-trash-move-modal/rbd-trash-move-modal.component.html	(date 1533458219000)
+++ src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-trash-move-modal/rbd-trash-move-modal.component.html	(date 1537454024000)
@@ -10,9 +10,9 @@
           novalidate>
       <div class="modal-body">
         <p>
-          <ng-container i18n>To move</ng-container>
+          <ng-container i18n>To move</ng-container>&nbsp;
           <kbd>{{ poolName }}/{{ imageName }}</kbd>&nbsp;
-          <ng-container i18n>to trash, click</ng-container>
+          <ng-container i18n>to trash, click</ng-container>&nbsp;
           <kbd i18n>Move Image</kbd>.&nbsp;
           <ng-container i18n>Optionally, you can pick a different expiration date.</ng-container>
         </p>
```